src/windows/audio.c

```diff
@@ -401,23 +425,37 @@ static bool audio_handle_device_change(MTY_Audio *ctx)
 
 uint32_t MTY_AudioGetQueued(MTY_Audio *ctx)
 {
-	return lrint((float) audio_get_queued_frames(ctx) / ((float) ctx->sample_rate / 1000.0f));
+	uint32_t queued = 0;
+	audio_get_queued_frames(ctx, &queued);
+	return lrint((float) queued / ((float) ctx->sample_rate / 1000.0f));
```

Comment:
queued here will be the buffer size, in the case of an error in `audio_get_queued_frames`. is that your intention?

Reply:
Yes, that is my intention. That is what the code was doing prior to my change as well. It is a reasonable choice as a sentinel value that communicates to calling code that the internal audio playback buffer is _full_ and so more data cannot be enqueued. _WHY_ it is full...internal error? actual buffer overrun? etc is (currently) not the concern of those who call `MTY_AudioGetQueuedFrames`. But it _is_ the concern of code within this file that uses `audio_get_queued_frames`, and the returned error code helps discern that.